packages/fast-components-react-base/src/divider/divider.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import * as ShallowRenderer from "react-test-renderer/shallow";
+import * as Adapter from "enzyme-adapter-react-16";
+import { configure, shallow } from "enzyme";
+import examples from "./examples.data";
+import { generateSnapshots } from "@microsoft/fast-jest-snapshots-react";
+import {
+    DividerProps,
+    DividerRoles,
+    IDividerClassNameContract,
+    IDividerHandledProps,
+    IDividerManagedClasses,
+    IDividerUnhandledProps
+} from "./divider";
+
+describe("divider", (): void => {
+    generateSnapshots(examples);
+});
+
+configure({adapter: new Adapter()});
+
+describe("divider unit-tests", (): void => {
+    const Component: React.ComponentClass<IDividerHandledProps & IDividerManagedClasses> = examples.component;
+    const managedClasses: IDividerClassNameContract = {
+        divider: "divider-class",
+    };
+
+    test("should return an object that includes all valid props which are note enumarated as handledProps", () => {
```

Comment:
looks like the misspelling of *not enumerated was carried over via CP, might want to check all of the tests you created for it

Reply:
Yeah this was a replacement for all of these from Nick's request for a more verbose statement.